rq/serializers.py

```diff
@@ -0,0 +1,80 @@
+from abc import ABCMeta
+from functools import partial
+from .exceptions import UnpickleError, DeserializationError
+
+try:
+    import cPickle as pickle
+except ImportError:  # noqa  # pragma: no cover
+    import pickle
```

Comment:
There's only `pickle` in Python 3, so I think we can eliminate the `import cPickle` bit.